### ISSUE TITLE
Redefine the power operator(**) in `Complex` class

### ISSUE THREAD

**Issue by Kapi2910** [yash-srivastava19/Elixir#5]
The current definition of the power (**) operator does not work because of the following line:
> for _ in range(other):
Since other (which is a complex number) is not countable, this line will throw an error. 

When raising one complex number to the power of another complex number the use of polar coordinates will make our work easier.

**Comment by yash-srivastava19** [comment]
For a barebones library, I only made the case where we can raise a complex number to a real number power. Raising one complex number to other is something I have little idea about. How do you think we should work on that?

**Comment by Kapi2910** [comment]
It is simple yet tedious. Would you okay with using the standard math library that python has in it? like using sines, cosines and pow function would make things would make it extremely Complex numbers simple (pun intended).

**Comment by yash-srivastava19** [comment]
Yeah that works. Can you explain some of the math behind it as well? I am a little curious about how we can implement that as well? I just remember the fact that $i^i$ is a real number 😄

**Comment by Kapi2910** [comment]
Yeah sure. 

We can do it is as follows:
<img width="798" alt="Screenshot 2023-08-11 at 4 53 07 PM" src="https://github.com/yash-srivastava19/Elixir/assets/63846417/0509059e-2886-4163-a8a4-60a8dc78ad59">

**Comment by yash-srivastava19** [comment]
I will try to get to it as soon as possible
